flang/runtime/inquiry.cpp

```diff
@@ -15,10 +15,26 @@
 #include "tools.h"
 #include "flang/Runtime/descriptor.h"
 #include <algorithm>
-#include <stdio.h>
 
 namespace Fortran::runtime {
 
+// Helper to store integer value in result[at].
+template <int KIND> struct StoreIntegerAt {
+  void operator()(const Fortran::runtime::Descriptor &result, std::size_t at,
+      std::int64_t value) const {
+    *result.ZeroBasedIndexedElement<Fortran::runtime::CppTypeFor<
+        Fortran::common::TypeCategory::Integer, KIND>>(at) = value;
+  }
+};
+
+template <int KIND> struct GetIntegerAt {
```

Comment:
I do not think this helper is actually used.

Reply:
Yep.  I had it in there to help with debugging.  I'll remove it.